packages/openapi-generator/src/generator.ts

```diff
@@ -101,6 +102,26 @@ async function generateSDKSources(
   }
 }
 
+async function createApis(
+  serviceDir: string,
+  openApiDocument: OpenApiDocument
+) {
+  return openApiDocument.tags.map(tag => {
+    const matchedOperations = openApiDocument.operations.filter(o =>
+      o.tags?.includes(tag)
+    );
```

Comment:
Please, don't call it `o` ;)
```suggestion
    const matchedOperations = openApiDocument.operations.filter(operation =>
      operation.tags?.includes(tag)
    );
```